### ISSUE TITLE
Unable to open pbit file

### ISSUE THREAD

**Issue by sheerperversity** [stansw/vsts-open-in-powerbi#12]
Hello
I`m downloading the report and trying to open in in Power BI desktop. I`m using the same credentials for Power BI and VSTS, but getting an authorization error message.
However, I can access VSTS using standard Power BI connector, which seems to be different from the one you use
![capture](https://user-images.githubusercontent.com/32038249/37564184-0bbeebf4-2aaa-11e8-8240-3f1e59ea3032.JPG)
yours is first, the standard is second in the list attached.

**Comment by StanislawSwierc** [comment]
Thank you for reporting this problem! I looked into it and found the root cause. In January release of Power BI Desktop they changed functions this extension relies on, thus, two different data sources. Apparently I should be using **`VSTS.AccountContents`** instead of **`VSTS.Contents`**. function.

I will update extension today. In the meantime you should be able to unblock yourself by manually editing the query text in *.pbit file and change **`VSTS.Contents`** to **`VSTS.AccountContents`**. If you don't know where to change it please wait for an update.

```
let
    url = "https://stansw.visualstudio.com",
    collection = "",
    project = "vsts-open-in-powerbi",
    team = "vsts-open-in-powerbi Team",
    id = "d5349265-9c9d-4808-933a-c3d27b731657",

    // Create wrapper for VSTS.Contents function as a workaround for the static code analysis in Power BI Service.
    contents = (o) => VSTS.Contents(
        url, 
        [ 
            Version              = Record.FieldOrDefault(Record.FieldOrDefault(o, "Headers", []), "Referer", ""),
            IsRetry              = Record.FieldOrDefault(o, "IsRetry", false),
            ManualStatusHandling = Record.FieldOrDefault(o, "ManualStatusHandling", {}),
            Query                = Record.FieldOrDefault(o, "Query", []),
            RelativePath         = Record.FieldOrDefault(o, "RelativePath", null),
            Timeout              = Record.FieldOrDefault(o, "Timeout", null)
       ]),

    Source = Functions[WiqlRunFlatWorkItemQueryById](contents, url, [Collection = collection, Project = project, Team = team], id)
in
    Source
```

```
let
    url = "https://stansw.visualstudio.com",
    collection = "",
    project = "vsts-open-in-powerbi",
    team = "vsts-open-in-powerbi Team",
    id = "d5349265-9c9d-4808-933a-c3d27b731657",

    // Create wrapper for VSTS.Contents function as a workaround for the static code analysis in Power BI Service.
    contents = (o) => VSTS.AccountContents(
        url, 
        [ 
            Version              = Record.FieldOrDefault(Record.FieldOrDefault(o, "Headers", []), "Referer", ""),
            IsRetry              = Record.FieldOrDefault(o, "IsRetry", false),
            ManualStatusHandling = Record.FieldOrDefault(o, "ManualStatusHandling", {}),
            Query                = Record.FieldOrDefault(o, "Query", []),
            RelativePath         = Record.FieldOrDefault(o, "RelativePath", null),
            Timeout              = Record.FieldOrDefault(o, "Timeout", null)
       ]),

    Source = Functions[WiqlRunFlatWorkItemQueryById](contents, url, [Collection = collection, Project = project, Team = team], id)
in
    Source
```

**Comment by stansw** [comment]
I've just published version 2.1.0 which uses `VSTS.AccountContents` for all queries. By now everyone should be already on the recent Power BI Desktop, thus, this change should have low impact. for people.
